symphony-bdk-examples/bdk-core-examples/src/main/java/com/symphony/bdk/examples/MessageExampleMain.java

```diff
@@ -0,0 +1,90 @@
+package com.symphony.bdk.examples;
+
+import static com.symphony.bdk.core.config.BdkConfigLoader.loadFromSymphonyDir;
+
+import com.symphony.bdk.core.SymphonyBdk;
+import com.symphony.bdk.core.auth.exception.AuthInitializationException;
+import com.symphony.bdk.core.auth.exception.AuthUnauthorizedException;
+import com.symphony.bdk.core.config.exception.BdkConfigException;
+import com.symphony.bdk.core.service.stream.constant.AttachmentSort;
+import com.symphony.bdk.gen.api.model.MessageIdsFromStream;
+import com.symphony.bdk.gen.api.model.MessageMetadataResponse;
+import com.symphony.bdk.gen.api.model.MessageReceiptDetailResponse;
+import com.symphony.bdk.gen.api.model.MessageStatus;
+import com.symphony.bdk.gen.api.model.MessageSuppressionResponse;
+import com.symphony.bdk.gen.api.model.StreamAttachmentItem;
+import com.symphony.bdk.gen.api.model.V4ImportResponse;
+import com.symphony.bdk.gen.api.model.V4ImportedMessage;
+import com.symphony.bdk.gen.api.model.V4Message;
+import com.symphony.bdk.template.api.TemplateException;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Stream;
+
+/**
+ * This demonstrates a basic usage of the message service.
+ */
+public class MessageExampleMain {
+
+  public static final String STREAM_ID = "gXFV8vN37dNqjojYS_y2wX___o2KxfmUdA";
+  public static final String MESSAGE = "<messageML>Hello, World!</messageML>";
+  public static final String MESSAGE_ID = "LE1TxlLArVbpNKn-CXZuZn___ouXdRpnbQ";
+
+  public static final Instant SINCE = Instant.now().minus(Duration.ofHours(1));
+  public static final Instant TO = Instant.now();
+
+  public static void main(String[] args)
+      throws BdkConfigException, AuthUnauthorizedException, AuthInitializationException, TemplateException {
+    final SymphonyBdk bdk = new SymphonyBdk(loadFromSymphonyDir("config.yaml"));
+
+    //send a regular message
+    final V4Message regularMessage = bdk.messages().send(STREAM_ID, MESSAGE);
+
+    //use templates
+    final V4Message sentWithBuiltInTemplate = bdk.messages().send(STREAM_ID, "simpleMML",
+        Collections.singletonMap("message", "Hello from a built-in template!"));
+
+    final V4Message sendWithCustomTemplate = bdk.messages()
+        .send(STREAM_ID, "customTemplate.ftl", Collections.emptyMap());
+
+    //retrieve the details of existing messages
+    final V4Message message = bdk.messages().getMessage(MESSAGE_ID);
```

Comment:
Could we use `regularMessage.getMessageId()` instead of this static `MESSAGE_ID`?